libraries/AP_Baro/AP_Baro.h

```diff
@@ -112,8 +134,10 @@ class AP_Baro
     // pressure in Pascal
     float get_sealevel_pressure(float pressure) const;
 
-    // get scale factor required to convert equivalent to true airspeed
-    float get_EAS2TAS(void);
+    // get scale factor required to convert equivalent to true
+    // airspeed. This should only be used to update the AHRS value
+    // once per loop. Please use AP::ahrs().get_EAS2TAS()
+    float get_EAS2TAS(void) const;
```

Comment:
```suggestion
    float get_EAS2TAS_use_AHRS_instead(void) const;
```

.... or, perhaps, something more subtle

Reply:
added _